### PR TITLE
removed dead code and unused variables, updated .travis.yml

### PR DESCRIPTION
**Pull request by GregorioDiStefano**

### CODE REVIEW

**Comment by pasinskim** [comment]
This looks OK. One thing that needs more investigation is why server certificate error is not used anywhere, but this will change anyway with IAM service implemented.

**Comment by pasinskim** [comment]
@maciejmrowiec can you comment using those new checks in Travis?

**Comment by GregorioDiStefano** [comment]
@maciejmrowiec I'm stick with deadcode and varcheck for now, since structcheck doesnt support embedded/nested structs, and goconst seems to produce some false positives

File: .travis.yml [hunk 1]
```diff
@@ -77,8 +83,10 @@ before_script:
     # Test if all new commits are signed-off
     - ./check_signed_off.sh
 
-    # Perform static code analysys
+    # Perform static code analysis
     - go tool vet -unsafeptr=false `go list ./... | grep -v Godeps | sed -e "s,^,$GOPATH/src/,;s,.*:,,"`
+    - deadcode .
```
Comment (maciejmrowiec) [inline .travis.yml]: does those commands run on current dir or recursive?

**Comment by maciejmrowiec** [comment]
@pasinskim

File: .travis.yml (hunk 1 above)
Comment (GregorioDiStefano) [inline .travis.yml]: ouch, they dont run recursively, but since there aren't any modules (separate folders) in this folder, this isn't a problem, right?

Comment (maciejmrowiec) [inline .travis.yml]: no, this repo has one main dir (something to keep in ming when porting to different repo)

**Comment by GregorioDiStefano** [comment]
@pasinskim can i go ahead and merge this?

**Pull request closed without merging**